src/base-cookie.js

```diff
@@ -47,6 +67,19 @@ const set = (name, value, opts) => {
   document.cookie = str;
 };
 
+const getLastEventTime = (cookie) => {
```

Comment:
this function feels a little brittle in case the cookie somehow ends up getting a little malformed - thoughts on `try/catch` here?

Reply:
Good catch- I initially checked the behavior if the cookie string is malformed, and that almost always result in `strValue` being undefined. Splitting a string with no matches returns an array of the original string, and accessing a missing index returns undefined.

But for reasons I don't understand, `parseInt(undefined, 32) === 33790067563981` 😮 - I'll add more error handling for that